reminder/reminder.py

```diff
@@ -1,19 +1,19 @@
+import asyncio
+import collections
+import datetime
+import hashlib
 import inspect
 import logging
-import typing
-import hashlib
 import re
-import collections
-import discord
-import datetime
-import asyncio
+import typing
 
+import discord
```

Comment:
Please undo the formatting changes you made. This PR is about fixing breaking changes, not reformatting.